src/regression.js

```diff
@@ -18,6 +18,32 @@ module.exports = {
     console.log('yVals are:', yVals);
     console.log('localInitialMVals are:', localInitialMVals);
     /* eslint-enable no-console */
-    return n.uncmin(w => this.objective(w, xVals, yVals), localInitialMVals, 0.0001).solution;
+    return n.uncmin(w => this.objective(w, xVals, yVals), localInitialMVals, 0.001).solution;
   },
+
+  rSquared(xVals, yVals, betaVector) {
+    const SSresidual = n.sum(n.pow(n.sub(yVals, n.dot(xVals, betaVector)), 2));
+    const meanyVals = n.sum(yVals) / yVals.length;
+    const SStotal = n.sum(n.pow(n.sub(yVals, n.rep(n.dim(yVals), meanyVals)), 2));
+    return 1 - (SSresidual / SStotal);
+  },
+
+  tValue(xVals, yVals, betaVector) {
```

Comment:
* Again, parameter documentation
* Needs some unit testing
* `varError` and `varBeta`…why the `var` prefix? (see _src/index.js_ comments)
* The final iterations for `tValue` can be dropped to make this simpler:

  ```js
  return betaVector.map((betaVectorItem, index) => 
    betaVectorItem / Math.sqrt(varBeta[index][index]);
  ```